packages/terra-collapsible-menu-view/src/CollapsibleMenuView.module.scss

```diff
@@ -55,6 +59,11 @@
     &:first-child {
       padding-left: 0;
     }
+
+    :focus {
+      box-shadow: var(--terra-collapsible-menu-view-item-focus-box-shadow, none);
```

Comment:
Yes the box shadow is being defined in the fusion theme only

Reply:
So then can we just move the setting of this style to the fusion theme file?  Seems like it does not need a theme variable or to reside in the default theme.

Reply:
Oh it does affect some stuff actually, buttons will have a blue glow around it without removing the box shadow in the default theme
<img width="547" alt="Screenshot 2023-12-13 at 12 05 48 PM" src="https://github.com/cerner/terra-framework/assets/8482248/7645bc60-6813-4e54-a049-de85e9b82601">

Reply:
I am saying that you can place the orion-fusion-theme specific logic within that file.  It can remove the outline and add its unique box shadow without impacting other themes.